### DISCUSSION TITLE
Перестал работать сайт указанный в list-general.txt

### DISCUSSION THREAD

**Discussion by Fofl76** [Flowseal/zapret-discord-youtube#953] [General]
С блокировкой аниме сайтов решил добавить в список animego.org , всё работало. Сегодня пытаюсь запустить и сайт не открывается, хотя с впн всё работает, но мой сервер блочит просмотр. Почему сайт указанный в списке просто перестал открываться, хотя  ютуб и дискор работают с general.

**Comment by project-by-rocket** [comment]
пробуйте другие стратегии (батники), при которых заработает

**Comment by wontyy** [comment]
такая же проблема,добавил сайт escapefromtarkov.com и ничего не происходит.Пишет что выполнил 36 хостов(что было изначально при установке),хотя написано 37

**Comment by Flowseal** [comment]
Пробуйте другие батники
